controllers/admin/AdminAccessController.php

```diff
@@ -194,8 +194,18 @@ public function getCurrentProfileId()
         return (isset($_GET['id_profile']) && !empty($_GET['id_profile']) && is_numeric($_GET['id_profile'])) ? (int) $_GET['id_profile'] : 1;
     }
 
+    /**
+     * @param array $a module data
+     * @param array $b module data
+     *
+     * @return int
+     */
     protected function sortModuleByName($a, $b)
     {
+        if ((!array_key_exists('name', $a)) || (!array_key_exists('name', $b))) {
```

Comment:
Because strnatcmp can compare null / empty string. 
```php
$aName = isset($a['name']) ? $a['name'] : null;
$bName = isset($b['name']) ? $b['name'] : null;
return strnatcmp($aName, $bName);
```

Reply:
Makes sense 👍